src/question_baseselect.ts

```diff
@@ -1149,8 +1149,15 @@ export class QuestionSelectBase extends Question {
     }
     return strs.join(", ");
   }
-  private getItemDisplayValue(item: ItemValue): string {
-    if(item === this.otherItem && this.comment) return this.comment;
+  private getItemDisplayValue(item: ItemValue, val?: any): string {
+    if (item === this.otherItem) {
```

Comment:
if(item === this.otherItem) {
      return this.getStoreOthersAsComment() ? this.comment : val;
    }
    return item.locText.textOrHtml;